dist/style.css

```diff
@@ -282,9 +282,10 @@ input {
 }
 
 .fancy {
-  width: 60%;
+  width: 55%;
   margin-left: auto;
   margin-right: auto;
+  fill: red;
```

Comment:
I'm noticing that when we add SVG files as images, it's not possible to update it's color with CSS. However, I've figured out how to manipulate the color in the SVG file itself. So, here's the new plan:

- Get rid of this line of code.
- Leave the color of this horizontal rule alone for now.
- When we eventually merge the code from the `landon` branch in to a `staging` or `production` branch, I'll manually manipulate the color of this asset in its SVG file.

(Actually, what I'll really do is turn it into two separate files; one for a green version, and one for a purple version, because there is another mockup with this asset being used in a purple color.)

Reply:
I figured out how to change the color as well - I forgot to remove this but it's the correct color now in the updated version. Sorry about that! Removing this line though as forgot to do so.